src/components/box/box.stories.mdx

```diff
@@ -0,0 +1,68 @@
+import { Meta, Story, Preview, Props} from '@storybook/addon-docs/blocks';
+import Box from '.';
+import Form from '../form';
+import Textbox from '../../__experimental__/components/textbox';
+import Button from '../button';
+import StyledSystemProps from '../../../.storybook/utils/styled-system-props';
+
+<Meta title="Box" />
+
+# Box
+
+```javascript
+  import Box from 'carbon-react/lib/components/box';
+```
+
+## Box
+Basic component that gives access to a number of styling options. Should be used to achieve more precise layouts when required. Gives access to Spacing, Color, Layout and FlexBox attributes
+<Preview>
+  <Story name="default" parameters={{ info: { disable: true }}} >
+    <>
+      <Box m={3} px={3}>
+        <Form
+            onSubmit={ () => console.log('submit')}
+            leftSideButtons={ <Button onClick={ () => console.log('cancel') }>Cancel</Button> }
+            saveButton={ <Button buttonType='primary' type="submit">Save</Button> }
+            stickyFooter
+            >
+            <Textbox label="Textbox" />
+            <Textbox label="Textbox" />
+            <Hr mb={7} mt={7} />
+            <Textbox label="Textbox" />
+        </Form>
+      </Box>
+    </>
+  </Story>
+</Preview>
+
+## Flex
+<Preview>
+  <Story name="flex" parameters={{ info: { disable: true }}} >
+    <>
+      <Box m={3} px={3} display='flex' alignContent='center'>
+        <Form
+            onSubmit={ () => console.log('submit')}
+            leftSideButtons={ <Button onClick={ () => console.log('cancel') }>Cancel</Button> }
+            saveButton={ <Button buttonType='primary' type="submit">Save</Button> }
+            stickyFooter
+            >
+            <Textbox label="Textbox" />
+            <Textbox label="Textbox" />
+            <Hr mb={7} mt={7} />
+            <Textbox label="Textbox" />
+        </Form>
+      </Box>
+    </>
+  </Story>
+</Preview>
+
+## Spacing props
+When setting padding we recommend using either the `p`, `py` 
```

Comment:
Not sure if this is relevant in context of `Box`, looks like copied from `button.stories.mdx` :D

Reply:
Removed the comment and renamed